Formula/alpine.rb

```diff
@@ -11,8 +11,15 @@ class Alpine < Formula
     sha256 "8f52d4ebe9e445ec975cabdd74c4a48cef80eebb21f18c33644e99de1a6d2173" => :mountain_lion
   end
 
+  option "with-maildir", "Compile with support for Maildir format mailboxes"
+
   depends_on "openssl"
 
+  patch do
+    url "http://patches.freeiz.com/alpine/patches/alpine-2.20/maildir.patch.gz"
```

Comment:
@mikemcquaid: See the main comments on this PR. "Patches" in this project are more like optional components, and `patches.freeiz.com` *is* upstream. Might want to include a couple of them. (We could use the `resource` mechanism instead, too, if the `patch` semantics aren't appropriate.)